packages/bbui/src/FancyForm/FancyField.svelte

```diff
@@ -61,7 +65,7 @@
 
 <style>
   .fancy-field {
-    max-width: 400px;
+    max-width: var(--fancy-field-max-width);
```

Comment:
Should we keep the 400x as default?

Reply:
it is default - the variable in the background defaults to 400px